web/src/main/webapp/WEB-INF/classes/setup/sql/data/data-db-default.sql

```diff
@@ -766,5 +766,632 @@ INSERT INTO GUF_RatingCriteria (id, name, isinternal) VALUES (4, 'ServiceQuality
+                "query_string": {
+                  "query": "+linkProtocol:/OGC:WMS.*/"
+                }
+              }
+            }
+          }
+        },
+        "cl_hierarchyLevel.key": {
+          "meta": {
+            "vega": "arc"
+          },
+          "terms": {
+            "field": "cl_hierarchyLevel.key"
+          }
+        },
+        "tag.default": {
+          "meta": {
+            "vega": "arc"
+          },
+          "terms": {
+            "field": "tag.default",
+            "size": 10
+          }
+        }
+      }
+    },
+    "authentication": {
+      "enabled": false,
+      "signinUrl": "../../{{node}}/{{lang}}/catalog.signin",
+      "signoutUrl": "../../signout"
+    },
+    "cookieWarning": {
+      "cookieWarningMoreInfoLink": "",
+      "cookieWarningRejectLink": "",
+      "enabled": false
+    },
+    "editor": {
+      "allowRemoteRecordLink": true,
+      "appUrl": "../../{{node}}/{{lang}}/catalog.edit",
+      "createPageTpl": "../../catalog/templates/editor/new-metadata-horizontal.html",
+      "editorIndentType": "",
+      "enabled": true,
+      "facetConfig": {
+        "cl_status.key": {
+          "terms": {
+            "field": "cl_status.key",
+            "size": 15
+          }
+        },
+        "documentStandard": {
+          "terms": {
+            "field": "documentStandard",
+            "size": 10
+          }
+        },
+        "groupOwner": {
+          "terms": {
+            "field": "groupOwner",
+            "size": 10
+          }
+        },
+        "groupPublished": {
+          "terms": {
+            "field": "groupPublished",
+            "size": 10
+          }
+        },
+        "isHarvested": {
+          "terms": {
+            "field": "isHarvested",
+            "size": 2
+          }
+        },
+        "isPublishedToAll": {
+          "terms": {
+            "field": "isPublishedToAll",
+            "size": 2
+          }
+        },
+        "isTemplate": {
+          "terms": {
+            "field": "isTemplate",
+            "size": 5
+          }
+        },
+        "isValid": {
+          "terms": {
+            "field": "isValid",
+            "size": 10
+          }
+        },
+        "isValidInspire": {
+          "terms": {
+            "field": "isValidInspire",
+            "size": 10
+          }
+        },
+        "recordOwner": {
+          "terms": {
+            "field": "recordOwner",
+            "size": 10
+          }
+        },
+        "resourceType": {
+          "terms": {
+            "field": "resourceType",
+            "size": 20
+          }
+        },
+        "sourceCatalogue": {
+          "terms": {
+            "field": "sourceCatalogue",
+            "size": 15
+          }
+        }
+      },
+      "fluidEditorLayout": true,
+      "isFilterTagsDisplayed": false,
+      "isUserRecordsOnly": false,
+      "minUserProfileToCreateTemplate": ""
+    },
+    "footer": {
+      "enabled": true,
```

Comment:
Proposal to disable the footer by default, I find it kind of useless...

Reply:
no objection / not really feeling involved in the UI choices / don't quite care :)